### DISCUSSION TITLE
Some little easy fixes that would be awesome to see

### DISCUSSION THREAD

**Discussion by BlackCharon142** [TKMAX777/RemoteRelativeInput#5] [Ideas]
I've been using this program for about a month now and think it's perfect for what it is, but a few finishing touches would make it perfect, I actually tried to recreate this project with c# and it didn't go as planned(check it out [here ](https://github.com/BlackCharon142/RelativeMouseRDP)if you want, it's not as good as this project)
I was hoping you could recreate what I've done in the past month and make the program a little more user friendly

P.S I wrote the things I made in my project but you're welcome to take a look at mine if you want to
1-Hidden form border
2-Support for Fullscreen
3-A proper dashboard
4-being able to choose the window inside the dashboard
5-all in one client and server program
6-an installer that would install everything automatically

**Comment by TKMAX777** [comment]
Sorry. I hadn't noticed the comment on Discussion for a long time!

I've been a little busy with work lately and can't respond right away, but I think this program should at least have a better UI.

Full screen support is certainly desirable, but I don't know of any good way to do it!

I feel that the client-side implementation should be implemented using WPF, etc., rather than the current Win32 API, but I have not yet learned enough to support it.

I will rethink the implementation a bit more in the future when I have more free time.

Thanks for your comments and help!

**Comment by TKMAX777** [comment]
I have developed one that comes with an installation script and does not require an SSH connection.

https://github.com/TKMAX777/RDPRelativeInput
